### ISSUE TITLE
Config warning shows absolute path instead of project-relative path

### ISSUE THREAD

**Issue by pygarap** [rvben/rumdl#291]
## Summary

When rumdl emits this warning:

```

[config warning] Unknown global option in ..

```

it prints the **absolute** file path. This makes the output noisy and less portable. It would be more useful to print the path **relative to the project root** (or current working directory).

## Steps to reproduce

1. In a project, create a `rumdl.toml` (or your config file) with an invalid/unknown `[global]` option.
2. Run:
   - `rumdl check` (or any command that loads the config)
3. Observe the warning output.

## Actual result

The warning includes an absolute path to the config file (or the referenced file), for example:

- `/home/user/projects/my-repo/rumdl.toml`

## Expected result

The warning should include a project-relative path, for example:

- `rumdl.toml`
- `config/rumdl.toml`

## Why this matters

- Improves readability in terminals and CI logs.
- Makes output consistent across machines and environments.
- Avoids leaking local filesystem paths in shared logs.

**Comment by rvben** [comment]
Thanks for the feedback, @pygarap!

Fixed in v0.0.217. Config warning messages now use project-relative paths instead of absolute paths.
